src/theme/ods-chart-theme.ts

```diff
@@ -436,6 +436,8 @@ export class ODSChartsTheme {
     for (const axis of ['xAxis', 'yAxis']) {
       if (
         (!this.dataOptions[axis] || !this.dataOptions[axis].data) &&
+        'time' !== this.dataOptions[axis].type &&
```

Comment:
Not necessarily to do in this PR, but isn't it possible in the future to create/use an enum or a structure rather than a string to define "time" and "category"?

Reply:
[Mutualize main axis determination function](https://github.com/Orange-OpenSource/ods-charts/pull/45/commits/b6871b9db495e9a479ed870e0e71412c43390ac8)

Reply:
- [x] http://localhost:3000/test/html/
- [x] http://localhost:3000/test/examples/timeseries-chart/
- [x] http://localhost:3000/test/examples/bar-line-chart/
- [x] http://localhost:3000/test/examples/single-line-chart/
- [x] http://localhost:3000/docs/examples/single-line-chart
- [x] http://localhost:3000/docs/examples/multiple-line-chart
- [x] http://localhost:3000/docs/examples/bar-chart
- [x] http://localhost:3000/docs/examples/bar-line-chart